package.json

```diff
@@ -2,8 +2,21 @@
   "name": "@testing-library/preact",
   "version": "0.0.0-semantically-released",
   "description": "Simple and complete Preact DOM testing utilities that encourage good testing practices.",
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
   "types": "types/index.d.ts",
+  "exports": {
+    ".": {
+			"types": "./types/index.d.ts",
+			"import": "./dist/esm/index.js",
+			"browser": "./dist/cjs/index.js"
+		},
+		"./pure": {
+			"types": "./pure.d.ts",
+			"import": "./dist/esm/pure.js",
+			"browser": "./dist/cjs/pure.js"
```

Comment:
ESM output as `.js` with no `"type": "module"`? Any reason not to support Node here?

`"browser"` being CJS also looks a bit odd to me, though could be fine.

Reply:
Yes I realised a moment ago as well, we have to output these as mjs as well

Reply:
Ah, apologies then! 

Thanks for doing all of this!

Reply:
No need to apologize, was a very good shout, thanks 😄